lighthouse-core/test/fraggle-rock/gather/mock-driver.js

```diff
@@ -128,6 +159,20 @@ function mockDriverModule(driverProvider) {
   };
 }
 
+function mockTargetManagerModule() {
+  const targetManagerMock = createMockTargetManager();
+
+  /** @type {(instance: any) => (...args: any[]) => any} */
+  const proxyCtor = instance => function() {
+    // IMPORTANT! This must be a `function` not an arrow function so it can be invoked as a constructor.
```

Comment:
is this comment necessary or useful to future readers?

Reply:
IMO, yes. I can easily see a future mass change coming in and flipping this back without this comment